refactor: move ethersOverrides to Ethereum.ts

jtakalai opened the pull request:
also add them to StorageNodeRegistry, so now it seems all Polygon smart contract transactions are covered

The rest don't have overrides for now

harbu commented on src/connection/WebRtcEndpoint.js:
```diff
@@ -278,11 +279,48 @@ class WebRtcEndpoint extends EventEmitter {
+            const dc = this.dataChannels[address]
+            try {
+                if (dc.readyState === 'open') {
+                    if (dc.respondedPong !== undefined && !dc.respondedPong) {
```
I guess you could simplify condition to `dc.respondedPong === false`

jtakalai committed:
refactor: move ethersOverrides to Ethereum.ts

also add them to StorageNodeRegistry, so now it seems all Polygon smart contract transactions are covered

The rest don't have overrides for now

jtakalai committed:
fix: types

hpihkala reviewed (approved):
LGTM

jtakalai committed:
refactor: add ethers overrides to Data Union transactions too

also removed the explicit ethersOptions argument from the function for now, for consistency (and since it's now available through overrides if necessary)

hpihkala commented:
Please publish a new alpha when merged! 👍🏻
